### DISCUSSION TITLE
how to remove this button?

### DISCUSSION THREAD

**Discussion by shivam-devs** [refinedev/refine#3773] [Q&A] [answered]
![bug](https://user-images.githubusercontent.com/93398933/224126837-f9ac8ceb-2909-4e07-ace2-02076cf1a711.png)

**Comment by vinclou** [comment, marked as answer]
# This the extension to react-query, so in you App.tsx:
[react-query-docs](https://tanstack.com/query/v4/docs/react/devtools#options)

`Extend the options and hide the dev tools => but not sure why you need this hidden,
this is not shipped to prod.` DEV_ONLY tool


## Example
```jsx
					options={{
						reactQuery: {
							clientConfig: {
								defaultOptions: {
									queries: {
										staleTime: Timer.TwoMinutes
									}
								}
							},
							devtoolConfig: {
								initialIsOpen: true,
								position: 'bottom-right'
							}
						},
						syncWithLocation: true,
						warnWhenUnsavedChanges: false,
					}}
```
